v1alpha1/proto/grafeas.proto

```diff
@@ -762,6 +817,35 @@ message Occurrence {
 
   // Output only. The time this `Occurrence` was last updated.
   google.protobuf.Timestamp update_time = 10;
+
+  // Describes how this occurrence can be verified
+  oneof verification {
+    IntotoVerification intoto = 16;
+  }
+}
+
+// Includes information about how to verify an occurrence using in-toto
+// TODO(r2wend2): This copies some information from the occurrence itself need to reconcile before merging.
+message IntotoVerification {
+  string layout_note = 1;
```

Comment:
As mentioned in my original [review comment]( https://github.com/grafeas/grafeas/pull/89#issuecomment-356754636), we currently associate link metadata and layout metadata indirectly by step names, i.e. [upon final product verification](https://github.com/in-toto/in-toto/blob/0.2.0/in_toto/verifylib.py#L1289-L1293),  for each step in a the layout to be verified, we try to load `<step.threshold>` links identified by filenames `<step.name>.<authorized functionary keyid prefix>.link`.

We don't want to associate links directly with layouts, because a link does not need to know the layout it belongs to at creation time. Also links should be available for reuse with different layouts.